core/data/azure/infrastructure/epiphany-single-machine/data.yaml

```diff
@@ -0,0 +1,601 @@
+            file: ~/.ssh/epiphany-operations/id_rsa.pub
+            data:
+
+        vpn:
+          # Make SURE all of the data is correct below before enabling 'vpn'. It can also take 30 minutes to an hour to create.
+          enable: false
+          name: vpn-epi-bld-apps
+          # Only support RouteBased type of connection currently
+          type: RouteBased
+          active_active: false
+          # There are two types of 'sku' (Basic and Standard). Always use Standard so any client can use.
+          sku: Standard
+          # Address space that is required by Virtual Network Gateway. This address space must be inside of virtual_network.address_space and do not overlap with other subnets defined  
+          gateway_subnet_space: 10.1.2.0/24
+          client_configuration:
+          # This section is very important!
+          # You must specify the address_space that will be allocated to use on the VPN side of the conection that will
+          # be able to talk to your cluster.
+            address_space:
+              - 172.16.1.0/24
+            root_certificate:
+              # name is the name of the cert that was created for you by a trusted party OR a name you give a self-signed cert
+              name: EpiphanyRootCa
+              revoked_certificate:
+                name: SomeRevokedCert
+                thumbprint: bd0ef7d2c9XDFDFDFE9752169894d2
+              # public_cert_data is the actual base64 public key from your cert. Put it in 'as is'. The '|' tells yaml to use 'as is'.
+              public_cert_data: |
+                YOUR-BASE64-CLIENT-AUTH-PUBLIC-KEY
+
+          ip_configuration:
+            name: vpn-ip-config
+            public_ip:
+              output:
+                enable: true
+                sensitive: false
+              name: pip-vpn-epiphany
+              address_allocation: Dynamic
+              idle_timeout_in_minutes: 30
+
+              tags: *tags
+
+            # subnet_id is generated so use terraform variable
+            private_ip:
+              output:
+                enable: true
+                sensitive: false
+              address_allocation: Dynamic
+              # address only applies if 'address_allocation' is 'static'
+              address: 10.0.2.5
+
+        network_security_group:
+          enable: true
+          # Note: Could create an array of NSGs and reference them name + '_' + # (i.e., epiphany_nsg_001) maybe at somepoint if needed
+          name: security-nsg-epiphany
+          tags: *tags
+
+          rules:            
+            - name: ssh
+              description: Allow SSH
+              priority: 102
+              direction: Inbound
+              access: Allow
+              protocol: Tcp
+              source_port_range: "*"
+              destination_port_range: "22"
+              source_address_prefix: "*"
+              destination_address_prefix: "*"
+
+        virtual_network:
+          name: epiphany-bld-apps-vnet
```

Comment:
change vnet name to something like `epiphany-single-vnet`

Reply:
Fixed